.eslintrc.js

```diff
@@ -1,21 +1,53 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parserOptions: {
+    parser: 'babel-eslint',
+    ecmaVersion: 2017,
+    sourceType: 'module',
+  },
   env: {
     browser: true,
     node: true,
     'jest/globals': true,
-    es6: true
+    es6: true,
   },
-  extends: ['standard', 'plugin:jest/recommended'],
+  extends: [ 'standard', 'plugin:jest/recommended',  'plugin:vue/recommended' ],
   // required to lint *.vue files
   plugins: [
-    'html',
-    'jest'
+    'jest',
   ],
   // add your custom rules here
   rules: {
-    'operator-linebreak': ['error', 'after']
+    'operator-linebreak': [ 'error', 'after' ],
+    'array-bracket-spacing': [ 'error', 'always', { singleValue: true } ],
+    'no-unused-vars': [ 'error', { 'args': 'after-used' } ],
+    'computed-property-spacing': [ 'error', 'always' ],
+    'object-curly-spacing': [ 'error', 'always' ],
+    'operator-linebreak': [ 'error', 'after' ],
+    'comma-dangle': [ 'error', 'always-multiline' ],
+    'vue/max-attributes-per-line': [ 4, {
+      singleline: 4,
+    }],
+    'vue/attributes-order': [ 2, {
+      order: [
+        'LIST_RENDERING',
+        'CONDITIONALS',
```

Comment:
I think conditionals should be before list rendering

Reply:
it doesn't matter, cause v-for with v-if is kinda bug so I think the answer is no